hooks/006-add-gdm.chroot

```diff
@@ -138,3 +138,6 @@ ln -s ../writable/gdm3/custom.conf /etc/gdm3/custom.conf
 
 # Disable gnome-shell systemd
 rm -f /usr/lib/systemd/user/org.gnome.Shell*
+
+# Remove gnome-language-selector .desktop file
```

Comment:
Comment is just stating what the line does, would be better to describe why, e.g. "Stop gnome-language-selector showing in dash"

Reply:
Yep... I'll prepare a patch.